cora_description/urdf/thrusters.xacro

```diff
@@ -83,10 +83,10 @@
     <plugin name="cora_gazebo_thrust" filename="libusv_gazebo_thrust_plugin.so">
       <cmdTimeout>1.0</cmdTimeout>
       <robotNamespace>${namespace}</robotNamespace>
-      <xacro:include filename="$(find wamv_gazebo)/urdf/thruster_layouts/wamv_gazebo_thruster_config.xacro" />
+      <xacro:include filename="$(find cora_gazebo)/urdf/cora_gazebo_thruster_config.xacro" />
       <!-- Angle must be disabled for VORC -->
-      <xacro:wamv_gazebo_thruster_config name="left" enable_angle="false" />
-      <xacro:wamv_gazebo_thruster_config name="right" enable_angle="false" />
+      <xacro:cora_gazebo_thruster_config name="left" />
```

Comment:
Is the `enable_angle` for tilting the thrusters? I wasn't sure what they were for. I forced them to be false because we said we didn't want to allow customization.

Reply:
This parameter is for allowing azimuth rotation of the thrusters - see https://github.com/osrf/vrx/wiki/tutorials-thruster_articulation.  I turned off the thruster articulation with the `enableAngle` parameter in `cora_gazebo/urdf/cora_gazebo_thruster_config.xacro`